meta/pattern-states.md

```diff
@@ -1,13 +1,37 @@
-We probably need to define this; will crowdsource the brainstorming so we can standardize the status. Please add your thoughts and suggestions here.
+# Potential new nomenclature
+Below shows updated nomenclature around the maturity and state that each pattern can be in. Each pattern is either less or more mature (has a solution or doesn't), and goes through a multi-step review process.
 
-A combination of:
-
-* types of patterns: donut, idea, draft
-* draft states: in progress, draft complete, ready for review, ready for 2nd review, etc
-* # of reviews completed: none, one, two, ...
+<img src="/assets/img/patterns-steps-and-maturities.png" height="350" style="float:right" align="right" alt="Maturity and Review Steps as seen on Github">
 
+## Pattern Maturity
+* A - Donut (Lacks Solution)
+* B - Unproven (Ideas)
+* C - Proven (Draft)
```

Comment:
The words in parenthesis () were meant to explain the OLD terminology. That isn't clear. I will change to your suggestion